src/crates.rs

```diff
@@ -206,17 +241,13 @@ impl Crates {
         // build deps but ignore dev deps, since those are irrelevant for publishing.
         // We need to wait until we have our entire sub-tree to do this, built above.
 
-        fn populate_dependees<'a>(
-            all: &'a Crates,
-            tree: &mut HashMap<String, Details<'a>>,
-        ) {
+        fn populate_dependees<'a>(all: &'a Crates, tree: &mut HashMap<String, Details<'a>>) {
```

Comment:
maybe rename this to `populate_deps or populate_dependencies`

Reply:
I think this name was because it's the other way round; it's populating things that depend on the crates rather than things the crates depend on, hence dependees and not dependencies iirc :)

Reply:
ah :)